cypress/e2e/secureArea.cy.js

```diff
@@ -0,0 +1,21 @@
+/// <reference types="cypress" />
+
+describe('Secure Area page', () => {
+  before(() => {
+    cy.visit('/login');
+  });
+
+  it('should provide the possibility to log out', () => {
+    const username = 'tomsmith';
+    const password = 'SuperSecretPassword!';
+
+    cy.loginUser(username, password);
+
+    cy.checkAuthByHref('Logout')
+      .click();
+
+    cy.checkAuthByMessage('You logged out of the secure area!');
```

Comment:
```suggestion
    cy.assertSuccessMessageHasAppeared('You logged out of the secure area!');
```
I think it would have been named like this